CMakeLists.txt

```diff
@@ -49,6 +54,13 @@ add_compile_definitions(REALM_ENABLE_ENCRYPTION)
 add_compile_definitions(REALM_INSTALL_LIBEXECDIR)
 add_compile_definitions(REALM_BUILD_LIB_ONLY)
 
+# on Apple platforms we use the built-in CFRunLoop
+# everywhere else it's libuv, except UWP where it doesn't build
```

Comment:
And I assume that `UWP` is checked by `WINDOWS_STORE`

Reply:
Indeed